osu.Game/Online/Chat/DrawableLinkCompiler.cs

```diff
@@ -23,12 +25,21 @@ public partial class DrawableLinkCompiler : OsuHoverContainer
         /// <summary>
         /// Each word part of a chat link (split for word-wrap support).
         /// </summary>
-        public readonly List<Drawable> Parts;
+        public readonly SlimReadOnlyListWrapper<Drawable> Parts;
```

Comment:
The change of type here was just to limit outwards mutability, yeah?

Reply:
Yep. It's not used anywhere aside from tests either way, so felt safe,